src/Seq.Forwarder/Cli/Commands/RunCommand.cs

```diff
@@ -72,11 +74,11 @@ protected override int Run(TextWriter cout)
             Log.Logger = CreateLogger(config.Diagnostics.InternalLoggingLevel, config.Diagnostics.InternalLogPath);
 
             var builder = new ContainerBuilder();
-            builder.RegisterModule(new SeqForwarderModule(_storagePath.BufferPath, config));
+            builder.RegisterModule(new SeqForwarderModule(_storagePath.BufferPath, _listenUri.ListenUri, config));
```

Comment:
E.g. this `_listenUri.ListenUri` could default to `null`, allowing the use of `_listenUri.ListenUri ?? config.Api.ListenUri` as the argument. `run` could then still accept `--listen`, but use the config otherwise.